libssh2-sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libssh2-sys"
-version = "0.2.11"
+version = "0.3.0"
```

Comment:
I think this is just a new feature so this shouldn't need to get a major bump, right?

Reply:
Yeah, I'll fix this to be a minor bump